FATN_to_USB/tests/test_get_fatn_dropbox_link.py

```
@@ -0,0 +1,383 @@
+
+    @pytest.mark.unit
+    def test_feed_with_malformed_html(self, malformed_html):
+        """Test that parser handles malformed HTML gracefully."""
+        parser = GetFATNUrl()
+
+        # Should not raise exception
+        try:
+            parser.feed(malformed_html)
+            # Should still extract tinyurl if present
+            assert 'tinyurl' in parser.FATNurl.lower()
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)